experimental/traffic-portal/src/app/api/profile.service.ts

```diff
@@ -71,6 +71,33 @@ export class ProfileService extends APIService {
 		return this.get<Array<ResponseProfile>>(path).toPromise();
 	}
 
+	/**
+	 * Retrieves Profiles associated with a Parameter from the API.
+	 *
+	 * @param p Either a {@link ResponseParameter} or an integral, unique identifier of a Parameter, for which the
+	 * Profiles are to be retrieved.
+	 * @returns The requested Profile(s).
+	 */
+	public async getProfilesByParam(p: number| ResponseParameter): Promise<Array<ResponseProfile>> {
+		let id: number;
+		if (typeof p === "number") {
+			id = p;
+		} else {
+			if (!p.id || p.id < 0) {
```

Comment:
`!p.id || p.id <0` is equivalent to `p.id <= 0` because the only `true` value for `!p.id` is `0`, since its type won't allow any other "false-y" value. But also, I don't think it's your responsibility to make sure the caller is passing a valid ID; if that happens the request will fail and they'll get an appropriate error message.